index.js

```diff
@@ -27,6 +35,15 @@ app.get(BASE_API_PATH + "/providers", (req, res) =>{
 app.post(BASE_API_PATH + "/providers", (req, res) => {
     console.log(Date() + "- POST /providers");
     var provider = req.body;
+    db.insert(providers,(err)=>{
```

Comment:
The correct variable to insert the document must be "provider".

Reply:
ya hecho